week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -13,7 +13,7 @@ function mood() {
 
 function greaterThan10() {
   let num = 10;
-  let isBigEnough;
+  let isBigEnough = 10;
```

Comment:
Hi Bashir, 
Think you could use a boolean expression to check if `num` is bigger than `10`. Consider using the `<` symbol to check this statement 😄